swupd/create_manifests.go

```diff
@@ -52,12 +53,18 @@ func getOldManifest(path string) (*Manifest, error) {
 	return ParseManifestFile(path)
 }
 
-func initBundles(ui UpdateInfo, c config) ([]*Manifest, error) {
+func initBundles(ui UpdateInfo, c config, numWorkers int) ([]*Manifest, error) {
 	var wg sync.WaitGroup
-	workers := len(ui.bundles)
-	wg.Add(workers)
+	if numWorkers <= 0 {
+		numWorkers = runtime.NumCPU()
```

Comment:
I set the default to runtime.NumCPU() to be consistent with the default value used in other parts of the code.